From 3d4063d1a0d5d19ac5d481d7df2092ebb0b40f4e Mon Sep 17 00:00:00 2001
From: Hao Bui <hao.bui.yg@renesas.com>
Date: Tue, 11 Aug 2020 10:06:15 +0700
Subject: [PATCH 499/504] ARM: dts: r8a7742: Add fallback compat string for
 usbphy

The RZ/G1H SoC (R8A7742) use the same usbphy as RCar Gen2. Thus, add
common rcar-gen2-usb-phy compat string for it.

Signed-off-by: Hao Bui <hao.bui.yg@renesas.com>
---
 arch/arm/boot/dts/r8a7742.dtsi | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/r8a7742.dtsi b/arch/arm/boot/dts/r8a7742.dtsi
index 46f0e41..91fd952 100644
--- a/arch/arm/boot/dts/r8a7742.dtsi
+++ b/arch/arm/boot/dts/r8a7742.dtsi
@@ -800,7 +800,8 @@
 	};
 
 	usbphy: usb-phy@e6590100 {
-		compatible = "renesas,usb-phy-r8a7742";
+		compatible = "renesas,usb-phy-r8a7742",
+			     "renesas,rcar-gen2-usb-phy";
 		reg = <0 0xe6590100 0 0x100>;
 		clocks = <&mstp7_clks R8A7742_CLK_HSUSB>;
 		clock-names = "usbhs";
-- 
2.7.4

